controllers/api.js

```diff
@@ -216,6 +216,88 @@ exports.getGithub = async (req, res, next) => {
   }
 };
 
+/**
+ * GET /api/imgur
+ * Imgur API example.
+ */
+exports.getImgur = async (req, res, next) => {
+  try {
+    const clientId = process.env.IMGUR_CLIENT_ID;
+
+    if (!clientId) {
```

Comment:
I am not sure if this condition is ever going to trigger since we have a placeholder value in .env.example